.github/workflows/build_and_push_to_ecr.yml

```
@@ -43,8 +43,8 @@ jobs:
         run: |
           if aws ecr describe-images --repository-name ${{ secrets.ECR_REPOSITORY }} --image-ids imageTag=${{ github.sha }} >/dev/null 2>&1; then
             echo "Image with tag ${{ github.sha }} already exists in ECR."
-            echo "::warning::Image already exists in ECR, stopping build workflow with error status to prevent release workflow from running."
-            exit 1
+            echo "::warning::Image already exists in ECR, stopping build workflow with success status as no further steps are required."
```

```suggestion
            echo "Image already exists in ECR, stopping build workflow with success status as no further steps are required."
```
Recommend removing the `::warning::` annotation. 
(but cool to know about that feature! more info here: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message )